unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/Tests/BIWTranslateGizmosShould.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+public class BIWTranslateGizmosShould
+{
+    private GameObject mockedGizmosGameObject;
+    private GameObject mockedEntityGameObject;
+    private BIWTranslateGizmos gizmos;
+    private BIWGizmosAxis axis;
+
+    [UnitySetUp]
+    protected IEnumerator SetUp()
```

Comment:
```suggestion
    [SetUp]
    protected void SetUp()
```
Same as the other comment.